.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  env: {
+    commonjs: true,
```

Comment:
D'après la doc: "commonjs - CommonJS global variables and CommonJS scoping (use this for browser-only code that uses Browserify/WebPack)."

Reply:
Ajouté automatiquement par la commande "eslint --init" pour un projet node.js Corrigé.